src/algebra/ordered_field.lean

```diff
@@ -387,16 +307,16 @@ lt_of_not_ge $ λ hn, not_le_of_gt hl $ one_div_le_one_div_of_le h hn
 lemma lt_of_one_div_lt_one_div_of_neg {a b : α} (h : b < 0) (hl : 1 / a < 1 / b) : b < a :=
 lt_of_not_ge $ λ hn, not_le_of_gt hl $ one_div_le_one_div_of_le_of_neg h hn
 
-lemma one_div_le_of_one_div_le_of_pos {a b : α} (ha : a > 0) (h : 1 / a ≤ b) : 1 / b ≤ a :=
+lemma one_div_le_of_one_div_le_of_pos {a b : α} (ha : 0 < a) (h : 1 / a ≤ b) : 1 / b ≤ a :=
```

Comment:
order again

Reply:
I think the lemma name is still wrong here.

Reply:
I don't understand why I missed this one. Searched for `_of_pos ` and `of_neg `, renamed more lemmas.